frontend/lib/src/components/widgets/AudioInput/formatTime.ts

```diff
@@ -15,13 +15,27 @@
  */
 
 const formatTime = (timeMs: number): string => {
-  const date = new Date(timeMs)
+  const seconds = Math.floor(timeMs / 1000)
+  const minutes = Math.floor(seconds / 60)
+  const hours = Math.floor(minutes / 60)
 
-  return date.toLocaleTimeString(undefined, {
-    minute: "2-digit",
-    second: "2-digit",
-    timeZone: "UTC",
-  })
+  const remainingSeconds = seconds % 60
+  const remainingMinutes = minutes % 60
+
+  if (seconds < 60) {
```

Comment:
If less than an hour, we like to display minutes and seconds (ex: `05:17`). If greater than an hours, we like to display hours,min,sec (ex: `01:12:05`)